geoips/compare_outputs.py

```diff
@@ -214,95 +214,46 @@ def images_match(output_product, compare_product, fuzz="5%"):
     bool
         Return True if images match, False if they differ
     """
-    out_diffimg = get_out_diff_fname(compare_product, output_product)
     exact_out_diffimg = get_out_diff_fname(
         compare_product, output_product, flag="exact_"
     )
-
-    call_list = [
-        "compare",
-        "-verbose",
-        "-quiet",
-        "-metric",
-        "ae",
-        "-fuzz",
-        fuzz,
-        output_product,
-        compare_product,
-        out_diffimg,
-    ]
-
-    exact_call_list = [
-        "compare",
-        "-verbose",
-        "-quiet",
-        "-metric",
-        "ae",
-        output_product,
-        compare_product,
-        exact_out_diffimg,
-    ]
-
-    LOG.info("**Running %s", " ".join(call_list))
-    fullimg_ret = subprocess.run(
-        call_list, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+    from PIL import Image
+    import numpy as np
+    from pixelmatch.contrib.PIL import pixelmatch
+
+    LOG.info("**Comparing output_product vs. compare product")
+    out_img = Image.open(output_product)
+    comp_img = Image.open(compare_product)
+    diff_img = Image.new(mode="RGB", size=comp_img.size)
+    diff_arr = np.abs(np.array(comp_img) - np.array(out_img))
```

Comment:
This fails if the images are different sizes - probably just need to catch this.

  File "geoips/geoips/compare_outputs.py", line 228, in images_match
    diff_arr = np.abs(np.array(comp_img) - np.array(out_img))
ValueError: operands could not be broadcast together with shapes (2463,2178,4) (2467,2178,4)

Reply:
This looks to be handled already.